plenum/test/view_change/test_view_change_n_minus_f_quorum.py

```diff
@@ -0,0 +1,27 @@
+from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.spy_helpers import get_count
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import ensure_view_change
+from plenum.test.helper import stopNodes
+
+
+def test_view_change_n_minus_f_quorum(nodeSet, up, looper):
+    """
+    Check that quorum n - f is used for view change 
+    """
+
+    # Quorum for view change is expected to be f + 1
```

Comment:
In this test we check that 3 nodes are able to do a view change. It just checks that quorum is <=n-f. 
Probably in another test we could stop 2 nodes and make sure that no view change happens?